<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnPendaftaran">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>358</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Pendaftaran Karyawan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBukuTamu">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>357</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Buku Tamu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPemesanan">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>357</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Pemesanan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLaporan">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>358</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Laporan Karyawan</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
